packages/flutter/test/material/chip_test.dart

```diff
@@ -14,6 +14,7 @@ import 'package:flutter/scheduler.dart';
 import '../rendering/mock_canvas.dart';
 import '../widgets/semantics_tester.dart';
 import 'feedback_tester.dart';
```

Comment:
Before and after:
<img width="450" alt="chips_before" src="https://user-images.githubusercontent.com/7915388/103947231-a115da80-50ec-11eb-950b-77b771e5fc24.png">

<img width="450" alt="chips_after" src="https://user-images.githubusercontent.com/7915388/103947234-a2470780-50ec-11eb-92f7-eaa7650e4f8e.png">